src/node_buffer.cc

```diff
@@ -857,7 +857,7 @@ void Compare(const FunctionCallbackInfo<Value> &args) {
   args.GetReturnValue().Set(val);
 }
 
-int32_t FastCompare(v8::Local<v8::Value>,
+int32_t FastCompare(FastApiTypedArray<Value>,
```

Comment:
I think you all these backwards. We need to **remove** FastApiTypedArray, not add more

Reply:
thanks